apps/website/src/content/docs/tag.mdx

```diff
@@ -13,28 +13,37 @@ thumbnail: Tag
 
```

Comment:
`TagMainExample` can probably show a tag with `onClose` too, instead of two with `onRemove`.

Reply:
I don't think so. Tags' main purpose is not to be used with `onClick`.